exporter/opentelemetry-exporter-otlp-proto-grpc/tests/test_otlp_metrics_exporter.py

```diff
@@ -802,9 +802,9 @@ def test_shutdown_wait_last_export(self):
             # pylint: disable=protected-access
             self.assertTrue(self.exporter._export_lock.locked())
             # delay is 4 seconds while the default shutdown timeout is 30_000 milliseconds
-            start_time = time.time()
+            start_time = time_ns()
             self.exporter.shutdown()
-            now = time.time()
+            now = time_ns()
             self.assertGreaterEqual(now, (start_time + 30 / 1000))
```

Comment:
Either I don't understand the change or there is a new bug here. How can it be correct to change `start_time` and `now` from seconds to nanoseconds without changing the `30 / 1000`?

Secondly, does anyone understand what this assert is supposed to do? I understand it somehow like: "Ensure that the shutdown took at least 30 seconds.". If this is the case, shouldn't it be `30 * 1000` then originally and now `30 * 1000_000?

Reply:
Thanks for reporting, @LarsMichelsen let me see what can I do here... :v: